packages/components/webpack.config.js

```diff
@@ -85,7 +87,9 @@ module.exports = {
                     to: 'assets/scss/theme'
                 }
             ]
-        })
+        }),
+        new IgnorePlugin(/^\.\/locale$/, /moment$/),
+        // new BundleAnalyzerPlugin()
```

Comment:
Did you mean to keep this commented out?

Reply:
If we're not going to actively use it we should probably not keep it in the package.json.